bentoml/README.md

```diff
@@ -0,0 +1,25 @@
+# BentoML
+
+This task allows operations on BentoML services.
```

Comment:
Can you add a link to more info? I'm not familiar with this service.

Reply:
Ah, good call - I added a link to the projects repository.

Reply:
Which removed your lgtm, oops.